core/mondoo-windows-security.mql.yaml

```diff
@@ -646,8 +628,8 @@ queries:
             default: "Failure";
           }
     mql: |
-      auditpol.where(subcategory == props.auditpolAuditPolicyChange).list != []
-      auditpol.where(subcategory == props.auditpolAuditPolicyChange).all(inclusionsetting == props.auditpolSuccess || inclusionsetting == props.auditpolSuccessFailure)
+      auditpol.where(subcategoryguid == "6997984D-797A-11D9-BED3-505054503030").list != []
+      auditpol.where(subcategoryguid == "6997984D-797A-11D9-BED3-505054503030").all(inclusionsetting == props.auditpolSuccess || inclusionsetting == props.auditpolSuccessFailure)
```

Comment:
i think the guid value is wrong. it should be 0CCE922F-69AE-11D9-BED3-505054503030